tests/tester.cc

```diff
@@ -474,7 +474,7 @@ TEST_CASE("image-uri-spaces", "[issue-236]") {
     }
     REQUIRE(true == ret);
     REQUIRE(err.empty());
-    REQUIRE(!warn.empty());  // relative image path won't exist in tests/
+    REQUIRE(warn.empty());
```

Comment:
This test was relying on the buggy behaviour, i.e. a warning appearing because the image could be loaded (because it wasn't saved in the first place).